scripts/zones/The_Garden_of_RuHmet/bcnms/when_angels_fall.lua

```diff
@@ -62,8 +62,8 @@ end;
 	
 function onEventFinish(player,csid,option)
 -- print("bc finish csid "..csid.." and option "..option);
-  if(csid== 0x7d01)then
-    player:setPos(420,0,445,192);
-end
+    if(csid== 0x7d01)then
+        player:setPos(420,0,445,192);
+    end
  

```

Comment:
You didn't move the end. You changed the whietspace. Lua doesn't care about whats between words it is not a whitespace sensative laguage. It could be written as
```lua
if(csid== 0x7d01)then player:setPos(420,0,445,192); end
```
or
```lua
if(csid== 0x7d01)then
player:setPos(420,0,445,192);
end
```
and still have the same behavior (would make me cry to have to read, but it wouldn't stop it from working).